connector_exchange/models/exchange_backend/common.py

```diff
@@ -42,15 +33,8 @@ def select_versions(self):
         """
         return [('exchange_2010', 'Exchange 2010')]
 
+    # todo remove version, no needed in exchangelib
     version = fields.Selection(selection='select_versions', required=True)
-    certificate_location = fields.Char(
-        required=True,
-        default="/opt/exchange_ws/certificates/")
-    location = fields.Char(
```

Comment:
I must be blind but how/where do you configure the URL to find the exchange server?

Reply:
I cannot see something like
```
config = Configuration(server='mail.example.com', credentials=credentials)
account = Account(primary_smtp_address='john@example.com', config=config,
                  autodiscover=False, access_type=DELEGATE)
```
found in the documentation of the exchangelib

Reply:
I seems to me it is taken from the domain of the email of the impersonation account. I will recheck, but I was able to connect without providing the URL and the certificate.

Reply:
ok. my customer does not use external email address to use the connector. I will add an option to disable autodiscover feature and provide a config object instead.

Reply:
You are right. Maybe we can add a boolean field to indicate autodiscover or not.

Reply:
that's what I was thinking about. I will make a PR against yours :smile: